client-mvp-04/src/sass/elements/_dropdown.scss

```diff
@@ -0,0 +1,57 @@
+@import '../variables';
+
+.dropdown-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
Also, to demonstrate how select works without collapsing section let's add `height: 380px;` and remove these 3 lines.